CONTRIBUTING.md

```diff
@@ -73,6 +73,13 @@ file output still looks reasonable and execute the following command to update t
 ./tool/update_goldens.sh
 ```
 
+To update the Stable versions of the Golden files, switch to Flutter's current stable
```

Comment:
Lets add a TODO somewhere to verify you really are on stable when you say you are.

Reply:
Will do! I wasn't sure how we could reliably do that (without fetching from GitHub) and figured the build will fail on the PR (once we get the stable tests running) so it's not critical. I'll a TODO though.

Reply:
(Done! I added two, since we should probably do the same for ToT too).